applications/StructuralMechanicsApplication/python_scripts/assign_nodal_elements_to_nodes_process.py

```diff
@@ -0,0 +1,8 @@
+# Importing the Kratos Library
```

Comment:
I'd use the registry to avoid creating this Python layer.

Reply:
Okay, thinjnk this is old PR